run_benchmarks.sh

```diff
@@ -3,16 +3,16 @@ set -xe
 
 run_duration="${RUN_DURATION:-60}"
 
-export GOMAXPROCS=24
-export COHTTP_DOMAINS=24
-export HTTPAF_EIO_DOMAINS=24
-export RUST_CORES=24
+export GOMAXPROCS=1
+export COHTTP_DOMAINS=1
+export HTTPAF_EIO_DOMAINS=1
+export RUST_CORES=1
```

Comment:
https://github.com/ocaml-multicore/retro-httpaf-bench/blob/6d7aaeac4a7d00be05b5f81d5ceead14de4facac/run_benchmarks_mult.sh exists for multicore benchmarks, so I switched the number of domains back to 1 in this script.